test/input_gen/transLayer_v2.py

```diff
@@ -64,6 +64,37 @@ def transpose_(weight):
     new_params = [transpose_(params[0]), transpose_(params[1]), bias]
     yield from new_params
 
+@register_for_((torch.nn.GRUCell))
+def gru_translate(model):
+    params = [(name, tensor.detach()) for name, tensor in model.named_parameters()]
+    bias = ("bias", params[2][1] + params[3][1])
+
+    # hidden, input -> input, hidden
```

Comment:
the code says the equation: `hidden, input -> hidden, input^T`. Can you please check

Reply:
No.
In here the weight[0] is the name and weight[1] is the tensor which has the shape [hidden, input]
I wrap it with the square bracket to clarify it as a dimension.